tests/subsys/debug/coredump/src/main.c

```diff
@@ -39,7 +39,8 @@ __no_optimization void func_3(uint32_t *addr)
 	defined(CONFIG_BOARD_LONGAN_NANO) || \
 	defined(CONFIG_BOARD_QEMU_XTENSA) || \
 	defined(CONFIG_BOARD_RISCV32_VIRTUAL) || \
-	defined(CONFIG_SOC_FAMILY_INTEL_ADSP)
+	defined(CONFIG_SOC_FAMILY_INTEL_ADSP) || \
+	defined(CONFIG_SOC_FAMILY_INTEL_ISH)
```

Comment:
You can add the entry before the last one, so that there is no need to modify existing ones.

Reply:
change already as suggested. thanks.